play/wikirecent/mzcompose

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Is this `mzcompose` bash script necessary for every project or am I just copying something because other projects have it?

Reply:
it exists to make demo directories self-contained, in the same way that you added `bin/{up,down}` in this PR, instead of documenting the same functionality in root/bin/mzconduct :-)